src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -1,11 +1,6 @@
 package seedu.addressbook.data.person;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
+import java.util.*;
```

Comment:
Please follow the [Java Coding Standard](https://nus-cs2103-ay1819s2.github.io/cs2103-website/coding-standards/java/intermediate.html) and configure IntelliJ accordingly to avoid using star imports.

To do so go to preferences, under editor > code style. Choose Java, under the tab imports, change class count and names count to use with '*' to a high value, like 999.

Reply:
Thanks, applied.